src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -328,6 +328,12 @@ public static FormulaValue Date(IRContext irContext, NumberValue[] args)
             var month = (int)args[1].Value;
             var day = (int)args[2].Value;
 
+            // dates before 1900 get converted to post-1900
+            if (year < 1900)
```

Comment:
Do we really want to do this? This is a (bad, IMO) legacy behavior from Excel. I thought we decided to have the C# interpreter do the right thing.

Reply:
removed and updated test comment.